packages/apps/package.json

```diff
@@ -17,7 +17,6 @@
     "@oclif/test": "^2.2.20",
     "@types/chai": "^4",
     "@types/mocha": "^5",
-    "@types/node": "^10",
```

Comment:
Do we just not need these types at all?

Reply:
Correct. They appear to be included in at least Node >= 14.